build.cmd

```diff
@@ -188,8 +188,8 @@ dir /b "%__DotNetCliPath%\sdk"
 call "!VS%__VSProductVersion%COMNTOOLS!\VsDevCmd.bat"
 echo Commencing build of managed components for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
-set "__BuildLog=%__LogsDir%\msbuild_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-%_msbuildexe% /ConsoleLoggerParameters:ForceNoAlign "%__ProjectDir%\build.proj" %__MSBCleanBuildArgs% /p:RepoPath="%__ProjectDir%" /p:RepoLocalBuild="true" /p:RelativeProductBinDir="%__RelativeProductBinDir%" /p:NuPkgRid=win7-x64 /p:ToolchainMilestone=%__ToolchainMilestone% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=normal;LogFile="%__BuildLog%"
+set "__BuildLog=%__LogsDir%\Managed_%__BuildOS%__%__BuildArch%__%__BuildType%"
+%_msbuildexe% /ConsoleLoggerParameters:ForceNoAlign "%__ProjectDir%\build.proj" %__MSBCleanBuildArgs% /p:RepoPath="%__ProjectDir%" /p:RepoLocalBuild="true" /p:RelativeProductBinDir="%__RelativeProductBinDir%" /p:NuPkgRid=win7-x64 /p:ToolchainMilestone=%__ToolchainMilestone% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /flp:Summary;Verbosity=minimal;LogFile=%__BuildLog%.sum /flp1:warningsonly;logfile=%__BuildLog%.wrn /flp2:errorsonly;logfile=%__BuildLog%.err /flp3:Verbosity=normal;LogFile=%__BuildLog%.log /flp4:Verbosity=diagnostic;LogFile=%__BuildLog%.diag
 IF NOT ERRORLEVEL 1 (
```

Comment:
The actual issue on dotnet/CoreCLR#1211 looks like applying the verbosity level to test projects, together with the product. I'd argue that the right fix should have been to split the way you build product and test and pass appropriate verbosity levels. I'll let this drop - for now - because I'm going to work on splitting the build scripts in various ways and I don't know how much of the build scrips I'll preserve. However, I'll include a similar change on building the product in that refactoring.